ftest.sh

```diff
@@ -173,14 +174,18 @@ fi
 mkdir /var/run/daos_{agent,server}
 chown \$current_username -R /var/run/daos_{agent,server}
 chmod 0755 /var/run/daos_{agent,server}
-mkdir -p $DAOS_BASE
-ed <<EOF /etc/fstab
+if [ -f $DAOS_BASE/SConstruct ]; then
+    echo "No need to NFS mount $DAOS_BASE"
```

Comment:
(lint) This is actually an end quote, but due to next char it looks suspect. [SC1079]
(lint) Double quote to prevent globbing and word splitting. [SC2086]